src/_P020_Ser2Net.ino

```diff
@@ -24,7 +24,7 @@
 # define P020_RX_WAIT              PCONFIG(4)
 # define P020_SERIAL_CONFIG        PCONFIG(1)
 # define P020_SERIAL_PROCESSING    PCONFIG(5)
-# define P020_RESET_TARGET_PIN     CONFIG_PIN1
+# define P020_RESET_TARGET_PIN     PCONFIG(6)
```

Comment:
Why change the setting for this?
It is possible to change the labels of the standard pins.

Reply:
The user shoud decide if he want to reset the pin and in the case which pin.

Reply:
So they can select the "- None -" pin, right?
Or am I missing something here?
Is the `CONFIG_PIN1` perhaps also configured by the ESPEasySerial selection?
If so, I guess it would make sense to use `CONFIG_PIN3`

Reply:
exactly: it can also be "none", which should be default; but if someone really wants to reset something a pin can be selected